builder/dispatchers.go

```diff
@@ -261,7 +266,11 @@ func run(b *Builder, args []string, attributes map[string]bool, original string)
 	args = handleJsonArgs(args, attributes)
 
 	if !attributes["json"] {
-		args = append([]string{"/bin/sh", "-c"}, args...)
+		if runtime.GOOS != "windows" {
+			args = append([]string{"/bin/sh", "-c"}, args...)
+		} else {
+			args = append([]string{"cmd", "/S /C "}, args...)
```

Comment:
you sure the cmd line processor doesn't want /S and /C as separate args?   Maybe windows is ok with it, dunno.